Source/Scene/Model.js

```diff
@@ -2083,6 +2096,17 @@ define([
             drawFS = 'uniform vec4 czm_pickColor;\n' + drawFS;
         }
 
+        if (model._upgrade10To20) {
+            drawFS = ShaderSource.replaceMain(drawFS, 'non_gamma_corrected_main');
+            drawFS =
+                drawFS +
+                '\n' +
+                'void main() { \n' +
+                '    non_gamma_corrected_main(); \n' +
+                '    gl_FragColor.rgb = czm_gammaCorrect(gl_FragColor.rgb); \n' +
```

Comment:
@bagnell @lilleyse @emackey on a related note - does this fix any of the gamma correction issues that I believe the glTF community has brought up about differences between Cesium and other WebGL engines when rendering glTF models?